### PR TITLE
client exec auth: updates for 1.11

### PR DESCRIPTION
**Pull request by ericchiang**
/cc @awly 

ref https://github.com/kubernetes/kubernetes/issues/61796

### CODE REVIEW

**Comment by awly** [comment]
This somewhat overlaps with https://github.com/kubernetes/website/pull/8826
I can drop that PR if you'd like to update all in one go.

**Comment by ericchiang** [comment]
Since yours already has a lgtm let's just wait on that then I'll rebase
once it's merged.

File: content/en/docs/reference/access-authn-authz/authentication.md [hunk 1]
```diff
@@ -745,75 +745,55 @@ the binary `/home/jane/bin/example-client-go-exec-plugin` is executed.
...
-```
-KUBERNETES_EXEC_INFO='{
-  "apiVersion": "client.authentication.k8s.io/v1alpha1",
-  "kind": "ExecCredential",
-  "spec": {
-    "interactive": true
-  }
-}'
-```
+The executed command is expected to print an `ExceCredential` object to `stdout`. `k8s.io/client-go`
```
Comment (zacharysarah) [inline content/en/docs/reference/access-authn-authz/authentication.md]: Replace lines 753-754 with:
```
The executed command prints an `ExecCredential` object to `stdout`. `k8s.io/client-go`
authenticates against the Kubernetes API using the returned credentials in the `status`. 
```

File: content/en/docs/reference/access-authn-authz/authentication.md [hunk 2]
```diff
@@ -745,75 +745,55 @@ the binary `/home/jane/bin/example-client-go-exec-plugin` is executed.
...
+will then use the returned credentials in the `status` when authenticating against the Kubernetes API.
 
-When plugins are executed from an interactive session, `stdin` and `stderr` are directly
-exposed to the plugin so it can prompt the user for input for interactive logins.
+When run from an interactive session `stdin` is exposed directly to the plugin. Plugins should use a
```
Comment (zacharysarah) [inline content/en/docs/reference/access-authn-authz/authentication.md]: Replaces lines 756-758 with:
```
When run from an interactive session, `stdin` is exposed directly to the plugin. Plugins should use a
[TTY check](https://godoc.org/golang.org/x/crypto/ssh/terminal#IsTerminal) to determine if it's
appropriate to prompt a user interactively.
```

**Comment by zacharysarah** [comment]
@ericchiang Bump

**Comment by liggitt** [comment]
closing in favor of rebased version in https://github.com/kubernetes/website/pull/9154

/close

**Pull request closed without merging**